Code/GraphMol/ChemReactions/catch_tests.cpp

```diff
@@ -1198,5 +1202,15 @@ TEST_CASE("V3K rxn blocks") {
     CHECK(rxn->getNumReactantTemplates()==rxn2->getNumReactantTemplates());
     CHECK(rxn->getNumProductTemplates()==rxn2->getNumProductTemplates());   
   }
+     
+  SECTION("github5324") {
+    // Test sgroup in a ring - this example failed with improperr tail crossings
+    auto mol = "C-1-C-C-C-C-O-1 |Sg:n:4:n:ht|"_smarts;
+    MolOps::findSSSR(*mol);
+    auto mbk = FileParserUtils::getV3000CTAB(*mol, -1);
```

Comment:
Please add a correctness test:
```
```
```suggestion
    auto mbk = FileParserUtils::getV3000CTAB(*mol, -1);
    CHECK(mbk.find("ATOMS=(1 5) XBONDS=(2 4 5) XBHEAD=(1 4) XBCORR=(2 4 5)")!=std::string::npos);
```